resources/ext/build-scripts/gem-list.txt

```diff
@@ -5,3 +5,4 @@ locale 2.1.2
 gettext 3.2.2
 fast_gettext 1.1.2
 hiera-eyaml 2.1.0
+jrjackson 0.3.8
```

Comment:
Why not 0.4.5?

Reply:
I thought there was a reason, but looking through the changelog now I'm not seeing one. It might have had something to do with me making sure it worked with various JRuby versions. I'll update.